lib/src/main/java/com/diffplug/spotless/scala/ScalaFmtStep.java

```diff
@@ -39,23 +34,17 @@ public class ScalaFmtStep {
 	// prevent direct instantiation
 	private ScalaFmtStep() {}
 
-	private static final Pattern VERSION_PRE_2_0 = Pattern.compile("[10]\\.(\\d+)\\.\\d+");
-	private static final Pattern VERSION_PRE_3_0 = Pattern.compile("2\\.(\\d+)\\.\\d+");
-	private static final String DEFAULT_VERSION = "3.0.8";
+	private static final String DEFAULT_VERSION = "3.5.9";
```

Comment:
Technically speaking we can leave this as `3.0.0` but since I am working on scalafmt I thought that I may as well update it to the latest version.